code/meta-parent/pom.xml

```diff
@@ -290,7 +290,7 @@
             <dependency>
                 <groupId>mysql</groupId>
                 <artifactId>mysql-connector-java</artifactId>
-                <version>5.1.26</version>
```

Comment:
We don't actually use the MySQL driver.  Feel free to remove this dependency management reference.  We actually use the MariaDB driver.  This is because MariaDB is LGPL and MySQL is GPL.  We can't use the GPL driver for many legal reasons.